src/main/java/core/basesyntax/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitOperation;
+import core.basesyntax.strategy.operation.OperationHandler;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
+    private Map<FruitOperation.Operation,OperationHandler> operationOperationHandlerMap;
```

Comment:
The field 'operationOperationHandlerMap' should be final to ensure it's not modified after being initialized. This is part of making classes immutable when possible, which is a good practice.